russian/call-mom/call-mom.md

```diff
@@ -1,10 +1,10 @@
 Вы позвонили своей маме и рассказали о том, что происходит.
-Она вам сообщила, что Вас на самом деле похитила группа 
-преступников и погрузила в сон одного из них - так 
-называемого Архитектора. Как только она начала говорить зачем 
-- звонок оборвался
+Она вам сообщила, что Вас на самом деле похитила группа
+преступников и погрузила в сон одного из них - так
+называемого Архитектора. Как только она начала говорить зачем - звонок оборвался
+
+[Попытаться проснуться](../../zefir.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.